packages/app-btc/src/operations/signTxn/index.ts

```diff
@@ -140,7 +143,13 @@ export const signTxn = async (
   }
 
   forceStatusUpdate(SignTxnStatus.SIGN_TXN_STATUS_CARD);
+  const signedTransaction: string = createSignedTransaction({
+    inputs,
+    outputs: params.txn.outputs,
+    signatures,
+    derivationPath: params.derivationPath,
+  });
 
   logger.info('Completed');
-  return { signatures };
+  return { signedTransaction };
```

Comment:
```suggestion
  return { signedTransaction, signatures };
```

Reply:
why tho?

Reply:
To provide flexibility, we return the device result as well as the derived result.

1. `signatures`: What we get from device
2. `signedTransaction`: What we derive from the `signatures` and `params`

Reply:
added